contracts/modules/legacy/LegacyPooledStaking.sol

```diff
@@ -1445,10 +1405,10 @@ contract LegacyPooledStaking is IPooledStaking, MasterAwareV2 {
       }
 
       token().approve(address(tokenController()), trancheDeposit);
-      IStakingPool(stakingPoolAddress).depositTo(
+      tokenId = IStakingPool(stakingPoolAddress).depositTo(
```

Comment:
Important:

I changed  it to what i assume it's what's the intention:

use the same token for each tranche deposit so that the manager owns a single NFT instance.


@roxdanila 
@shark0der  let me know if you think it should be otherwise.